debian/copyright

```diff
@@ -1,14 +1,15 @@
-Copyright 2014-2017 the Comprehensive Kerbal Archive Network (CKAN) Authors.
-https://github.com/KSP-CKAN/CKAN/graphs/contributors
-
-You can use the CKAN and its associated files under the MIT license,
-reproduced below. This includes the right to sublicense under compatible
-licenses.
-
----
-
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: CKAN
+Upstream-Contact: http://webchat.esper.net/?channels=ckan
+Source: https://github.com/KSP-CKAN/CKAN/
+
+Files: *
+Copyright: 2014-2017, the Comprehensive Kerbal Archive Network (CKAN) Authors:
```

Comment:
We should probably update the 2017 endpoint to 2019 now.

Reply:
Ideally, instead of updating the file every time there's a change in a new year, the Makefile could read the date of the current git commit and plug that year into the package with `sed`. But that's enough of an ask that I might as well leave it set to the current year until somebody gets the urge to automate it.